radio/src/targets/taranis/diskio.cpp

```diff
@@ -978,7 +978,7 @@ void sdPoll10ms()
 
 FATFS g_FATFS_Obj;
 #if defined(LOG_TELEMETRY)
-FIL g_telemetryFile = {0};
+FIL g_telemetryFile;

```

Comment:
> radio\src\targets\horus\diskio.cpp:313:25: warning: missing braces around initializer for '_FDID' [-Wmissing-braces]

Anyway seemed like a "strange" way to init a struct.

And yes, Horus diskio also, same warning.  Only happens when building fw. with LOG_TELEMETRY defined.